src/Native/Unix/configure.cmake

```diff
@@ -362,6 +362,40 @@ check_prototype_definition(
 
 check_cxx_source_compiles(
     "
+    #include <stdlib.h>
+    #include <unistd.h>
+    #include <string.h>
+
+    int main()
+    {
+        char* path = strdup(\"abc\");
+        int ret = mkstemps(path, 3);
+        return 0;
+    }
+    "
+    HAVE_MKSTEMPS)
+
+check_cxx_source_compiles(
+    "
+    #include <stdlib.h>
+    #include <unistd.h>
+    #include <string.h>
+
+    int main()
+    {
+        char* path = strdup(\"abc\");
+        int ret = mkstemp(path);
+        return 0;
+    }
+    "
+    HAVE_MKSTEMP)
+
+if (NOT HAVE_MKSTEMPS AND NOT HAVE_MKSTEMP)
+    message(FATAL_ERROR "Cannot find mkstemp nor mkstemp on this platform.")
```

Comment:
This is good for now.  If we come across a platform that doesn't have either, we can write yet another fallback.